spynl/tests/conftest.py

```diff
@@ -14,7 +14,7 @@
                                        encode_date)
 
 
-@pytest.fixture(scope="session")
```

Comment:
@ev-agelos Did you check the impact on performance? IIRC app creation (down below) takes a lot of time, but I could  be wrong.